priv/protos/test_service_v3.proto

```diff
@@ -29,6 +29,11 @@ message TestRequest {
     }
 
     Location location = 2;
+    Token token = 3;
+  }
+
+  message Token {
```

Comment:
A nested payload is a message declaration inside of a message declaration, like
```
message A {
  B this = 0;
  message B {
    string message = 0;
  }
}
```

I'm not sure this effectively tests the parsing of nested messages

Reply:
`TestRequest.Payload` covers nested types tests.
`TestRequest.Token` covers following patterns.
```
message A {
  B b = 1;
  message B {
    C c = 1;
  }
  message C {
  }
}
```
In this pattern, Type-B and Type-C are both nested types in type-A, but B refers C.

Reply:
yup, `get_nested_types/2` is doing what I expected 👍